jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/Menu.java

```diff
@@ -20,6 +22,16 @@ public String hasPosition() {
         return String.format("top: %s; left: %s", core().getCssValue("top"), core().getCssValue("left"));
     }
 
+    @JDIAction("'{name}' has position from top")
+    public String hasTopPosition() {
```

Comment:
Wrong name.
If method has name has* that means that the result type is boolean, the same as is* 

Also it is a very bad idea to combine the string "top: ", typically we need a real number to use in autotests

Reply:
Corrected